crates/muvm/src/cli_options.rs

```diff
@@ -98,6 +99,11 @@ pub fn options() -> OptionParser<Options> {
         .argument("SERVER_PORT")
         .fallback(3334)
         .display_fallback();
+    let direct_x11 = long("direct-x11")
+        .short('x')
+        .help("Use direct X11 forwarding instead of sommelier + XWayland")
+        .flag(true, false);
```

Comment:
```suggestion
        .switch();
```

https://docs.rs/bpaf/latest/bpaf/parsers/struct.NamedArg.html#method.switch

Reply:
Fixed, thanks!